test/prism/ruby_api_test.rb

```diff
@@ -20,6 +20,18 @@ def test_ruby_api
       assert_equal_nodes ast2, ast3
     end
 
+    def test_parse_success?
+      assert Prism.parse_success?("1")
+      refute Prism.parse_success?("<>")
+
+      assert Prism.parse_success?("m //", verbose: false)
+      refute Prism.parse_success?("m //", verbose: true)
```

Comment:
Not about this PR specifically but I find this a bit confusing/surprising. It seems like action at a distance. Why does changing verbosity affect the parse state? 

I checked the docs for that option https://rubydoc.info/gems/prism/Prism#parse-class_method it doesn't mention anything about affecting behavior (other than behavior of whether or not to warn).

Reply:
So in this case `parse_success?` checks if there are no errors or warnings. Verbosity controls whether warnings are emitted. So by turning off verbosity, we can say `parse_success?` was successful because no warnings were emitted.